src/routes/environment/update.ts

```diff
@@ -18,7 +19,7 @@ interface Request {
       nodes: UpdatableFieldConfigurations;
       pins: UpdatableFieldConfigurations;
     };
-    locale?: 'nl' | 'en';
+    locale?: Locale;
```

Comment:
Shouldn't the Joi-schema also be replaced (with `localeSchema`)?

Reply:
Sure, found few more places to apply it